wiki/president_place_of_birth_tutorial.ipynb

```diff
@@ -1425,528 +1428,33 @@
-      "[439 epo]: TRAIN:[loss=0.030]\n",
-      "[440 epo]: TRAIN:[loss=0.030]\n",
-      "[441 epo]: TRAIN:[loss=0.030]\n",
-      "[442 epo]: TRAIN:[loss=0.030]\n",
-      "[443 epo]: TRAIN:[loss=0.030]\n",
-      "[444 epo]: TRAIN:[loss=0.030]\n",
-      "[445 epo]: TRAIN:[loss=0.030]\n",
-      "[446 epo]: TRAIN:[loss=0.030]\n",
-      "[447 epo]: TRAIN:[loss=0.030]\n",
-      "[448 epo]: TRAIN:[loss=0.030]\n",
-      "[449 epo]: TRAIN:[loss=0.030]\n",
-      "[450 epo]: TRAIN:[loss=0.030]\n",
-      "[451 epo]: TRAIN:[loss=0.030]\n",
-      "[452 epo]: TRAIN:[loss=0.030]\n",
-      "[453 epo]: TRAIN:[loss=0.030]\n",
-      "[454 epo]: TRAIN:[loss=0.030]\n",
-      "[455 epo]: TRAIN:[loss=0.030]\n",
-      "[456 epo]: TRAIN:[loss=0.030]\n",
-      "[457 epo]: TRAIN:[loss=0.030]\n",
-      "[458 epo]: TRAIN:[loss=0.030]\n",
-      "[459 epo]: TRAIN:[loss=0.030]\n",
-      "[460 epo]: TRAIN:[loss=0.030]\n",
-      "[461 epo]: TRAIN:[loss=0.030]\n",
-      "[462 epo]: TRAIN:[loss=0.030]\n",
-      "[463 epo]: TRAIN:[loss=0.030]\n",
-      "[464 epo]: TRAIN:[loss=0.030]\n",
-      "[465 epo]: TRAIN:[loss=0.030]\n",
-      "[466 epo]: TRAIN:[loss=0.030]\n",
-      "[467 epo]: TRAIN:[loss=0.030]\n",
-      "[468 epo]: TRAIN:[loss=0.030]\n",
-      "[469 epo]: TRAIN:[loss=0.030]\n",
-      "[470 epo]: TRAIN:[loss=0.030]\n",
-      "[471 epo]: TRAIN:[loss=0.030]\n",
-      "[472 epo]: TRAIN:[loss=0.030]\n",
-      "[473 epo]: TRAIN:[loss=0.030]\n",
-      "[474 epo]: TRAIN:[loss=0.030]\n",
-      "[475 epo]: TRAIN:[loss=0.030]\n",
-      "[476 epo]: TRAIN:[loss=0.030]\n",
-      "[477 epo]: TRAIN:[loss=0.030]\n",
-      "[478 epo]: TRAIN:[loss=0.030]\n",
-      "[479 epo]: TRAIN:[loss=0.030]\n",
-      "[480 epo]: TRAIN:[loss=0.030]\n",
-      "[481 epo]: TRAIN:[loss=0.030]\n",
-      "[482 epo]: TRAIN:[loss=0.030]\n",
-      "[483 epo]: TRAIN:[loss=0.030]\n",
-      "[484 epo]: TRAIN:[loss=0.030]\n",
-      "[485 epo]: TRAIN:[loss=0.030]\n",
-      "[486 epo]: TRAIN:[loss=0.030]\n",
-      "[487 epo]: TRAIN:[loss=0.030]\n",
-      "[488 epo]: TRAIN:[loss=0.030]\n",
-      "[489 epo]: TRAIN:[loss=0.030]\n",
-      "[490 epo]: TRAIN:[loss=0.030]\n",
-      "[491 epo]: TRAIN:[loss=0.030]\n",
-      "[492 epo]: TRAIN:[loss=0.030]\n",
-      "[493 epo]: TRAIN:[loss=0.030]\n",
-      "[494 epo]: TRAIN:[loss=0.030]\n",
-      "[495 epo]: TRAIN:[loss=0.030]\n",
-      "[496 epo]: TRAIN:[loss=0.030]\n",
-      "[497 epo]: TRAIN:[loss=0.030]\n",
-      "[498 epo]: TRAIN:[loss=0.030]\n",
-      "[499 epo]: TRAIN:[loss=0.030]\n",
-      "[500 epo]: TRAIN:[loss=0.030]\n",
-      "Finished Training\n",
-      "CPU times: user 584 ms, sys: 44.4 ms, total: 629 ms\n",
-      "Wall time: 610 ms\n"
+      "CPU times: user 832 ms, sys: 32 ms, total: 864 ms\n",
+      "Wall time: 872 ms\n"
      ]
     }
    ],
    "source": [
-    "from metal.label_model import LabelModel\n",
+    "from snorkel.labeling.model import LabelModel\n",
```

Comment:
We also need to update the text content to show we are moving from metal back to snorkel.

Reply:
Updated. Thanks.